runtime/src/accounts_db.rs

```diff
@@ -2743,13 +3610,21 @@ impl AccountsDB {
         }
     }
 
+    pub fn store_cached(&self, slot: Slot, accounts: &[(&Pubkey, &Account)]) {
+        self.do_store(slot, accounts, self.caching_enabled);
+    }
+
     /// Store the account update.
-    pub fn store(&self, slot: Slot, accounts: &[(&Pubkey, &Account)]) {
+    pub fn store_uncached(&self, slot: Slot, accounts: &[(&Pubkey, &Account)]) {
         // If all transactions in a batch are errored,
         // it's possible to get a store with no accounts.
         if accounts.is_empty() {
             return;
         }
+        self.do_store(slot, accounts, false);
+    }
+
+    fn do_store(&self, slot: Slot, accounts: &[(&Pubkey, &Account)], is_cached_store: bool) {
```

Comment:
nits `fn store(` because there is no `store` in this scope anymore (yeah, before my review comment, it existed.); only `store_cached` and `store_uncached` now.